framework/src/Volo.Abp.AspNetCore.Components.WebAssembly.BasicTheme/BasicThemeBundleContributor.cs

```diff
@@ -4,12 +4,12 @@ namespace Volo.Abp.AspNetCore.Components.WebAssembly.BasicTheme
 {
     public class BasicThemeBundleContributor : IBundleContributor
     {
-        public void AddScripts(BundleContext context)
+        public void AddScripts(BundleContext context, BundleParameterDictionary parameters)
```

Comment:
We should add BundleParameterDictionary into the BundleContext to not introduce breaking change (which was the purpose of the BundleContext)